src/etc/inc/console.inc

```diff
@@ -384,6 +384,7 @@ EOD;
         $config['interfaces']['lan']['if'] = $lanif;
         $config['interfaces']['lan']['enable'] = true;
 
+        $config['system']['webgui']['antilockout_interface'] = 'lan';
```

Comment:
as said in the issue I'd prefer a per-interface field instead of a loose list maintained somewhere else.

Reply:
User view: I only use WAN and/or OPTX and I still want anti-lockout. What do I do on the console? A single WAN will also lock itself out...